app/controllers/miq_ae_class_controller.rb

```diff
@@ -2067,24 +2067,24 @@ def process_ae_ns(ae_ns, task)
   def get_form_vars
     @ae_class = MiqAeClass.find_by(:id => @edit[:ae_class_id])
     # for class add tab
-    @edit[:new][:name] = params[:name].blank? ? nil : params[:name] if params[:name]
-    @edit[:new][:description] = params[:description].blank? ? nil : params[:description] if params[:description]
-    @edit[:new][:display_name] = params[:display_name].blank? ? nil : params[:display_name] if params[:display_name]
+    @edit[:new][:name] = params[:name].presence if params[:name]
```

Comment:
Uh, all of those `foo.presence if foo` are useless, right? As in, `foo.presence if foo` is always the same as just `foo.presence`